src/Emogrifier/CssInliner.php

```diff
@@ -275,59 +174,6 @@ private function getHeadElement()
         return $this->domDocument->getElementsByTagName('head')->item(0);
     }
 
-    /**
-     * Creates a DOM document from the given HTML and stores it in $this->domDocument.
-     *
-     * The DOM document will always have a BODY element and a document type.
-     *
-     * @param string $html
-     *
-     * @return void
-     */
-    private function createUnifiedDomDocument($html)
-    {
-        $this->createRawDomDocument($html);
-        $this->ensureExistenceOfBodyElement();
-    }
-
-    /**
-     * Creates a DOMDocument instance from the given HTML and stores it in $this->domDocument.
-     *
-     * @param string $html
-     *
-     * @return void
-     */
-    private function createRawDomDocument($html)
-    {
-        $domDocument = new \DOMDocument();
-        $domDocument->encoding = 'UTF-8';
```

Comment:
`AbstractHtmlProcessor` does not set `encoding`.  Should it?

Reply:
Good catch! No, because `loadHTML` will overwrite the encoding anyway. I'll create a PR for removing that line.

Reply:
#661